monkey/monkey_island/cc/setup/mongo/mongo_setup.py

```diff
@@ -48,7 +49,6 @@ def register_mongo_shutdown_callback(mongo_db_process: MongoDbProcess):
 def connect_to_mongodb(timeout: float):
     _wait_for_mongo_db_server(MONGO_URL, timeout)
     _assert_mongo_db_version(MONGO_URL)
-    mongo_connector.connect_dal_to_mongodb()
```

Comment:
What replaces the call to connect?

Reply:
Nothing. This call is left over from when we were using an ORM. Each repository is passed a mongo client via dependency injection and makes its own connections.